### PR TITLE
[Audit Fix] - Add crabv2 events

### PR DESCRIPTION
**Pull request by KMKoushik**
Added events for CrabV2 and Migration

### CODE REVIEW

File: packages/hardhat/contracts/strategy/CrabMigration.sol [hunk 1]
```diff
@@ -83,6 +83,9 @@ contract CrabMigration is Ownable {
     }
 
     event ClaimAndWithdraw(address indexed user, uint256 crabAmount);
+    event DepositV1Shares(address indexed user, uint256 crabV1Amount);
+    event ClaimV2Shares(address indexed user, uint256 carbAmount);
```
Comment (aleone) [inline packages/hardhat/contracts/strategy/CrabMigration.sol:87]: carb loading!

(change to crab)

**Review by aleone — APPROVED** [review]
1 typo to fix, but lgtm otherwise

File: packages/hardhat/contracts/strategy/CrabMigration.sol (hunk 1 above)
Comment (KMKoushik) [reply packages/hardhat/contracts/strategy/CrabMigration.sol:87]: Lol crab.

**Review by alpinechicken — APPROVED** [review]
lgtm

**Pull request merged**